<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_rest_message">
    <sys_rest_message action="INSERT_OR_UPDATE">
        <access>public</access>
        <authentication_type>no_authentication</authentication_type>
        <basic_auth_password/>
        <basic_auth_profile/>
        <basic_auth_user/>
        <description/>
        <name>ChatGPT</name>
        <oauth2_profile/>
        <protocol_name/>
        <rest_endpoint>https://api.openai.com/v1/completions</rest_endpoint>
        <sys_class_name>sys_rest_message</sys_class_name>
        <sys_created_by>admin</sys_created_by>
        <sys_created_on>2023-07-23 11:28:25</sys_created_on>
        <sys_id>2b67e30897103110098ef2a3f153afce</sys_id>
        <sys_mod_count>1</sys_mod_count>
        <sys_name>ChatGPT</sys_name>
        <sys_package display_value="Incident Management" source="x_816691_inciden_0">7581019d978031103e74fed11153af8f</sys_package>
        <sys_policy/>
        <sys_scope display_value="Incident Management">7581019d978031103e74fed11153af8f</sys_scope>
        <sys_update_name>sys_rest_message_2b67e30897103110098ef2a3f153afce</sys_update_name>
        <sys_updated_by>admin</sys_updated_by>
        <sys_updated_on>2023-07-23 11:51:29</sys_updated_on>
        <use_basic_auth>false</use_basic_auth>
        <use_mutual_auth>false</use_mutual_auth>
    </sys_rest_message>
</record_update>
